mentoring-app/src/app/services/auth.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@angular/core';
+import { LoginData } from '../models/user.model';
+import { Router } from '@angular/router';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthService {
+
+  constructor(private router: Router) { }
```

Comment:
You can follow the following approach to practice alternative class providers here:
[(https://angular.io/guide/dependency-injection-in-action#supply-a-custom-provider-with-inject)]